playbooks/common/openshift-cluster/upgrades/v3_10/upgrade_control_plane.yml

```diff
@@ -45,6 +46,56 @@
   - set_fact:
       pre_upgrade_complete: True
 
+- import_playbook: label_nodes.yml
+
+# To upgrade, we need masters to be capable of signing certificates
```

Comment:
If a cluster is using an external CA (no key to sign) and upgrades with the default of bootstrapping enabled, will we abort? For the external CA use case we will need to make sure that bootstrapping is off until we have the pieces in place to support that..

Reply:
Yeah, it's in my check list up at the top.